### PR TITLE
Add support for file fixture available in Rails 5

### PR DESCRIPTION
**Pull request by morgoth**

### CODE REVIEW

**Comment by samphippen** [comment]
@morgoth can you rebase this against master and see if it starts passing?

**Comment by morgoth** [comment]
@samphippen It fails rubocop check - now method has too many lines. Not sure what to do about it?

**Comment by morgoth** [comment]
@samphippen I fixed rails5 failures, so now it only fails by rubocop check.
Should I disable this check for this method, change global rubocop config or do you have better idea?

**Comment by JonRowe** [comment]
@morgoth you can disable that check for that method.

**Comment by morgoth** [comment]
The build is green now :)

File: spec/rspec/rails/configuration_spec.rb [hunk 1]
```diff
@@ -252,4 +252,10 @@ def in_inferring_type_from_location_environment
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
   end
+
+  if ::Rails::VERSION::STRING > '5'
+    it "#file_fixture_path defaults to 'spec/fixtures/files'" do
```
Comment (samphippen) [inline spec/rspec/rails/configuration_spec.rb]: @morgoth this it statement doesn't scan cleanly.

Can you change it to `it "has a default #file_fixture_path of 'spec/fixtures/files'"`?

File: features/step_definitions/additional_cli_steps.rb [hunk 2]
```diff
@@ -15,3 +16,9 @@
     pending "ActiveJob is not available"
   end
 end
+
+Given /file fixture is available/ do
```
Comment (samphippen) [inline features/step_definitions/additional_cli_steps.rb]: Could you change this to `/file fixtures are available/`

**Comment by samphippen** [comment]
Few grammar comments, if you could patch those up, I'll merge this in.

**Pull request merged**